oojs.md

```diff
@@ -7,15 +7,24 @@ Define a `Playlist` class. It should have...
 - An `addSong` (Function) method that adds a song to the `songs` property
 
 ```js
-Your answer goes here
+class Playlist{
+	constructor(title,songs){
+		this.title = title,
+		this.songs = [songs],
```

Comment:
If the `songs` parameter is already and array, no need to wrap `songs` in brackets on the right side of this assignment. Otherwise, it might create a nested array.